Source/Core/DiscIO/VolumeCreator.cpp

```diff
@@ -211,16 +209,14 @@ static IVolume* CreateVolumeFromCryptedWiiImage(IBlobReader& _rReader, u32 _Part
 EDiscType GetDiscType(IBlobReader& _rReader)
 {
 	CBlobBigEndianReader Reader(_rReader);
-	u32 WiiMagic = Reader.Read32(0x18);
-	u32 WiiContainerMagic = Reader.Read32(0x60);
+	u32 WiiMagic = Reader.Read32(0x18); // Disc layout magic (Wii)
+	//u32 WiiContainerMagic = Reader.Read32(0x60); Debugger Hook
```

Comment:
Please don't leave code commented out like this, even if it's for debugging purposes.